### ISSUE TITLE
Impossible to parse BLEU score!

### ISSUE THREAD

**Issue by masonreznov** [facebookresearch/XLM#288]
Hello @glample @Celebio 
I am running the Unsupervised machine translation from a pretrained cross lingual langauge model in google colab. The language model is succesfully trained, while the MT training breaks with the following error:
`
WARNING - 04/18/20 06:30:16 - 1:31:36 - Impossible to parse BLEU score! ""
INFO - 04/18/20 06:30:16 - 1:31:36 - BLEU ./dumped/enmn_mlm_comparable/UNsupMT/hypotheses/hyp7.en-mn.test.txt ./dumped/enmn_mlm_comparable/UNsupMT/hypotheses/ref.en-mn.test.txt : -1.000000
/bin/sh: 1: /content/drive/My: not found
/pytorch/aten/src/ATen/native/cuda/LegacyDefinitions.cpp:19: UserWarning: masked_fill_ received a mask with dtype torch.uint8, this behavior is now deprecated,please use a mask with dtype torch.bool instead.
/pytorch/aten/src/ATen/native/cuda/LegacyDefinitions.cpp:19: UserWarning: masked_fill_ received a mask with dtype torch.uint8, this behavior is now deprecated,please use a mask with dtype torch.bool instead.
/pytorch/aten/src/ATen/native/cuda/LegacyDefinitions.cpp:19: UserWarning: masked_fill_ received a mask with dtype torch.uint8, this behavior is now deprecated,please use a mask with dtype torch.bool instead.
/pytorch/aten/src/ATen/native/cuda/LegacyDefinitions.cpp:19: UserWarning: masked_fill_ received a mask with dtype torch.uint8, this behavior is now deprecated,please use a mask with dtype torch.bool instead.
/pytorch/aten/src/ATen/native/cuda/LegacyDefinitions.cpp:19: UserWarning: masked_fill_ received a mask with dtype torch.uint8, this behavior is now deprecated,please use a mask with dtype torch.bool instead.
/pytorch/aten/src/ATen/native/cuda/LegacyDefinitions.cpp:19: UserWarning: masked_fill_ received a mask with dtype torch.uint8, this behavior is now deprecated,please use a mask with dtype torch.bool instead.
/pytorch/aten/src/ATen/native/cuda/LegacyDefinitions.cpp:19: UserWarning: masked_fill_ received a mask with dtype torch.uint8, this behavior is now deprecated,please use a mask with dtype torch.bool instead.
WARNING - 04/18/20 06:30:39 - 1:31:59 - Impossible to parse BLEU score! ""
INFO - 04/18/20 06:30:39 - 1:31:59 - BLEU ./dumped/enmn_mlm_comparable/UNsupMT/hypotheses/hyp7.mn-en.test.txt ./dumped/enmn_mlm_comparable/UNsupMT/hypotheses/ref.mn-en.test.txt : -1.000000
INFO - 04/18/20 06:30:39 - 1:31:59 - epoch -> 7.000000
INFO - 04/18/20 06:30:39 - 1:31:59 - valid_en-mn_mt_ppl -> 37129.505730
INFO - 04/18/20 06:30:39 - 1:31:59 - valid_en-mn_mt_acc -> 4.182061
INFO - 04/18/20 06:30:39 - 1:31:59 - valid_en-mn_mt_bleu -> -1.000000
INFO - 04/18/20 06:30:39 - 1:31:59 - valid_mn-en_mt_ppl -> 18670.284138
INFO - 04/18/20 06:30:39 - 1:31:59 - valid_mn-en_mt_acc -> 5.391411
INFO - 04/18/20 06:30:39 - 1:31:59 - valid_mn-en_mt_bleu -> -1.000000
INFO - 04/18/20 06:30:39 - 1:31:59 - test_en-mn_mt_ppl -> 38666.477615
INFO - 04/18/20 06:30:39 - 1:31:59 - test_en-mn_mt_acc -> 3.936057
INFO - 04/18/20 06:30:39 - 1:31:59 - test_en-mn_mt_bleu -> -1.000000
INFO - 04/18/20 06:30:39 - 1:31:59 - test_mn-en_mt_ppl -> 19451.364352
INFO - 04/18/20 06:30:39 - 1:31:59 - test_mn-en_mt_acc -> 5.150026
INFO - 04/18/20 06:30:39 - 1:31:59 - test_mn-en_mt_bleu -> -1.000000
INFO - 04/18/20 06:30:39 - 1:31:59 - __log__:{"epoch": 7, "valid_en-mn_mt_ppl": 37129.50573025578, "valid_en-mn_mt_acc": 4.182060914651184, "valid_en-mn_mt_bleu": -1, "valid_mn-en_mt_ppl": 18670.284138244577, "valid_mn-en_mt_acc": 5.3914112811883115, "valid_mn-en_mt_bleu": -1, "test_en-mn_mt_ppl": 38666.47761534584, "test_en-mn_mt_acc": 3.936056838365897, "test_en-mn_mt_bleu": -1, "test_mn-en_mt_ppl": 19451.364351510263, "test_mn-en_mt_acc": 5.1500256767661945, "test_mn-en_mt_bleu": -1}
INFO - 04/18/20 06:30:39 - 1:31:59 - Not a better validation score (3 / 10).
`

The problem is related with the white space in "My Drive", which leads the script to break
`
/bin/sh: 1: /content/drive/My: not found
`
Further, the line number 20 of /src/evaluation/evaluator.py
`assert os.path.isfile(BLEU_SCRIPT_PATH)`
is not giving any assertion error either.

**Comment by masonreznov** [comment]
Solved the issue by adding
`command = f'"{BLEU_SCRIPT_PATH}"' + ' %s < %s' `
in line number 558 of src/evaluation/evaluator.py
